backend/api/views/friendView.py

```diff
@@ -54,8 +54,52 @@ def friends_api(request, id):
             return Response({"detail":"node not ready, no remote request"}, status=403)
     else:
         # remote
-        # only allow get from remote
-        if request.method == 'GET':
+        # only allow get, delete from remote
+        if request.method != 'POST':
             return processFriendsApi(request, id)
         else:
-            return Response({"detail": "This method does not accept remote request."}, status=405)
\ No newline at end of file
+            return Response({"detail": "This method does not accept remote request."}, status=405)
+
+
+@api_view(['GET'])
+def author_as_friend(request):
+    request_host = request.META.get('HTTP_ORIGIN')
+    print("request host: ",request_host)    
+
+    if request_host==None:
+        # return Response({"detail":"no X-Request-Host header"}, status=400)
+        return Response({"detail":"no origin header"}, status=400)
+    else:
+        request_host += '/'
+
+    # request host?
+    if isHostFront(request_host):
+        # local
+        true_url = request.META.get('HTTP_X_URL')
+        print("url: ",true_url)
```

Comment:
remove print

Reply:
I would recommend keeping these prints, there are more works around these backends, it's easier to debug